tensorboard/plugins/debugger_v2/debug_data_multiplexer.py

```diff
@@ -36,9 +37,41 @@
 # ones are either repetitions of the earlier ones or caused by the earlier ones.
 DEFAULT_PER_TYPE_ALERT_LIMIT = 1000
 
+# Default interval between successive calls to `DebugDataReader.update()``.
+DEFAULT_RELOAD_INTERVAL_SEC = 60
 
-def run_in_background(target):
-    """Run a target task in the background.
+
+class Timer(object):
```

Comment:
This is a very thin abstraction over `threading.Event` and IMHO I think it'd be simpler to just use the underlying APIs directly (it's also slightly confusing as named, since `threading.Timer` exists but is a different thing entirely).

All this abstraction really provides is tying together the calls to `clear()` and `wait()`, but `sleep()` is only called in a single place so inlining the two calls would be easy.  See also comments below.

Reply:
OK. Done removing this Timer class and its associated unit tests. Thanks for the suggestion.